ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -310,9 +319,10 @@ def advance(self, env: SubprocessEnvManager) -> int:
                 if changed:
                     self.trainers[brain_name].reward_buffer.clear()
 
-        time_start_step = time()
-        new_step_infos = env.step()
-        delta_time_step = time() - time_start_step
+        with hierarchical_timer("env_step"):
```

Comment:
The decorator is more concise and doesn't need the whitespace change, but this is probably better for parts of the code where we're making calls to an abstract class.